pytest_picked.py

```diff
@@ -64,15 +68,20 @@ def _affected_tests():
     """
     raw_output = _get_git_status()
 
+    re_list = [item.replace('.', '\.').replace('*', '.*')
```

Comment:
Since we're using Black to enforce the code style, you should use double quotes in this line. I added Lambda Lint to help us with it.

Reply:
also changed.